custom/icds_reports/tests/agg_tests/outputs/agg_awc_2017-04-01_1_sorted.csv

```diff
@@ -1,4 +1,4 @@
+st1,All,All,All,All,2017-04-01,26,130,,,,,,,,,,,,,,,,,12,,338,497,,135,0,,,,,,,,50,84,609,5,11,1,0,0,0,0,0,0,0,0,0,0,2,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,9,5,7,,2,3,7,9,85,13,3,yes,,,,,,1,1,1,2,4,10,1315,418,428,,50,84,609,19,7,19,7,7,,,,9,177,743,2,1,1,0,1,0,0,0,0,0,0,1,0,270,91,62,3,134,0,,,6,5,9,16,0,84,50,84,50,64,93,64,93,229,223,229,223,54,65,54,65,142,122,142,122
```

Comment:
All these new added columns. I will remove in PR only

Reply:
You mean in this PR

Reply:
yes